src/components/modern/LargeTokenValueView.js

```diff
@@ -45,15 +49,16 @@ type Props = {|
  * Large (& stylized) component to display token value.
  */
 function LargeTokenValueView({ value, symbol, style }: Props) {
+  const fiatCurrency = useFiatCurrency();
   if (!value) return null;
 
   return (
     <Container style={style}>
       {/* TokenValue & TokenSymbol are wrapped in plain RN Text in order to make baseline work */}
       <RNText>
-        <TokenValue>{formatTokenValueWithoutSymbol(value, symbol, { stripTrailingZeros: true })}</TokenValue>
-        <Spacing w={spacing.extraSmall} />
-        <TokenSymbol>{symbol}</TokenSymbol>
+        <FiatValue>{formatFiat(value, fiatCurrency, { skipCents: true })}</FiatValue>
```

Comment:
Seems like `formatFiat` is not working, since on your sceenshots it displays zero value. `formatFiat` is legacy code, pls use `formatFiatValue` from `utils/format` with `getAssetValueInFiat` from `utils/rates`.

Reply:
Better yet try using `FiatValueView` with custom `style` + `getAssetValueInFiat`